packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2231,139 +2231,139 @@ export type DeclarativeIncomingCall = IncomingCall;
+    MessageDelivered: React_2.JSX.Element;
+    MessageEdit: React_2.JSX.Element;
+    MessageFailed: React_2.JSX.Element;
+    MessageRemove: React_2.JSX.Element;
+    MessageResend: React_2.JSX.Element;
+    MessageSeen: React_2.JSX.Element;
+    MessageSending: React_2.JSX.Element;
+    OptionsCamera: React_2.JSX.Element;
+    OptionsMic: React_2.JSX.Element;
+    OptionsSpeaker: React_2.JSX.Element;
+    ParticipantItemMicOff: React_2.JSX.Element;
+    ParticipantItemOptions: React_2.JSX.Element;
+    ParticipantItemOptionsHovered: React_2.JSX.Element;
+    ParticipantItemScreenShareStart: React_2.JSX.Element;
+    HoldCallContextualMenuItem: React_2.JSX.Element;
+    HoldCallButton: React_2.JSX.Element;
+    ResumeCall: React_2.JSX.Element;
+    SendBoxSend: React_2.JSX.Element;
+    SendBoxSendHovered: React_2.JSX.Element;
+    VideoTileMicOff: React_2.JSX.Element;
+    DialpadBackspace: React_2.JSX.Element;
+    SitePermissionsSparkle: React_2.JSX.Element;
+    SitePermissionCamera: React_2.JSX.Element;
+    SitePermissionMic: React_2.JSX.Element;
+    SitePermissionCameraDenied: React_2.JSX.Element;
+    SitePermissionMicDenied: React_2.JSX.Element;
+    UnsupportedEnvironmentWarning: React_2.JSX.Element;
+    BrowserPermissionDeniedError: React_2.JSX.Element;
+    VideoTilePinned: React_2.JSX.Element;
+    VideoTileMoreOptions: React_2.JSX.Element;
+    VideoTileScaleFit: React_2.JSX.Element;
+    VideoTileScaleFill: React_2.JSX.Element;
+    PinParticipant: React_2.JSX.Element;
+    UnpinParticipant: React_2.JSX.Element;
+    SplitButtonPrimaryActionCameraOn: React_2.JSX.Element;
+    SplitButtonPrimaryActionCameraOff: React_2.JSX.Element;
+    SplitButtonPrimaryActionMicUnmuted: React_2.JSX.Element;
+    SplitButtonPrimaryActionMicMuted: React_2.JSX.Element;
+    VerticalGalleryLeftButton: React_2.JSX.Element;
+    VerticalGalleryRightButton: React_2.JSX.Element;
+    ControlButtonVideoEffectsOption: React_2.JSX.Element;
+    ConfigurationScreenVideoEffectsButton: React_2.JSX.Element;
+    CaptionsIcon: React_2.JSX.Element;
+    CaptionsOffIcon: React_2.JSX.Element;
+    CaptionsSettingsIcon: React_2.JSX.Element;
+    ChangeSpokenLanguageIcon: React_2.JSX.Element;
+    ChangeCaptionLanguageIcon: React_2.JSX.Element;
+    ContextMenuCameraIcon: React_2.JSX.Element;
+    ContextMenuMicIcon: React_2.JSX.Element;
+    ContextMenuSpeakerIcon: React_2.JSX.Element;
 };
 
 // @public
 export const DEFAULT_COMPOSITE_ICONS: {
-    EditBoxCancel: JSX.Element;
-    EditBoxSubmit: JSX.Element;
-    MessageDelivered: JSX.Element;
-    MessageEdit: JSX.Element;
-    MessageFailed: JSX.Element;
-    MessageRemove: JSX.Element;
-    MessageSeen: JSX.Element;
-    MessageSending: JSX.Element;
-    ParticipantItemOptions: JSX.Element;
-    ParticipantItemOptionsHovered: JSX.Element;
-    SendBoxSend: JSX.Element;
-    SendBoxSendHovered: JSX.Element;
+    EditBoxCancel: JSX.Element | React_2.JSX.Element;
```

Comment:
This might cause a bit of a headache for the next beta and stable ARB reviews :/ Is it worth leaving api-extractor at its current version?